docs/integrations/cloudwatch/index.rst

```diff
@@ -0,0 +1,18 @@
+Amazon CloudWatch and Aiven
+===========================
+
+`Amazon CloudWatch (AWS) <https://aws.amazon.com/cloudwatch/>`_ is an AWS monitoring service that helps to observe your applications and infrastructure resources. Aiven provides integrations to enable you to to include an Aiven service data in your AWS CloudWatch.
+
+CloudWatch metrics
+------------------
+
+You can send the metrics from any or all of your Aiven services to CloudWatch. You can specify the namespace where your metrics should be send or one will be created for you. 
+
+Find out :doc:`how to send your Aiven service metrics to AWS CloudWatch </docs/integrations/cloudwatch/cloudwatch-metrics>`.
+
+CloudWatch logs
+---------------
+
+You can send any of your Aiven service logs to your Amazon CloudWatch logs service. 
+
+Check out our tutorial on :doc:`how to send your Aiven service logs to AWS CloudWatch </docs/integrations/cloudwatch/cloudwatch-logs>`.
```

Comment:
This doesn't seem to have linked properly in the preview at https://deploy-preview-961--developer-aiven-io-preview.netlify.app/docs/integrations/cloudwatch/index.html

Reply:
Oh you are right. I forgot to commit some files and ended up loosing it. fixed now!